ui/ducks/send/send.js

```diff
@@ -354,19 +355,37 @@ export const initializeSendState = createAsyncThunk(
     // the getMetaMaskAccounts selector. getTargetAccount consumes this
     // selector and returns the account at the specified address.
     const account = getTargetAccount(state, fromAddress);
-    // Initiate gas slices work to fetch gasPrice estimates. We need to get the
-    // new state after this is set to determine if initialization can proceed.
-    await thunkApi.dispatch(fetchBasicGasEstimates());
 
-    await getGasFeeEstimatesAndStartPolling();
-    const {
-      gas: { basicEstimateStatus, basicEstimates },
-    } = thunkApi.getState();
     // Default gasPrice to 1 gwei if all estimation fails
-    const gasPrice =
-      basicEstimateStatus === BASIC_ESTIMATE_STATES.READY
-        ? getGasPriceInHexWei(basicEstimates.average)
-        : '0x1';
+    let gasPrice = '0x1';
+    let basicEstimateStatus = BASIC_ESTIMATE_STATES.LOADING;
+    let gasEstimatePollToken = null;
+
+    if (Boolean(process.env.SHOW_EIP_1559_UI) === false) {
+      // Initiate gas slices work to fetch gasPrice estimates. We need to get the
+      // new state after this is set to determine if initialization can proceed.
+      await thunkApi.dispatch(fetchBasicGasEstimates());
+      const {
+        gas: { basicEstimates, basicEstimateStatus: apiBasicEstimateStatus },
+      } = thunkApi.getState();
+
+      basicEstimateStatus = apiBasicEstimateStatus;
+
+      if (basicEstimateStatus === BASIC_ESTIMATE_STATES.READY) {
+        gasPrice = getGasPriceInHexWei(basicEstimates.average);
+      }
+    } else {
+      // Instruct the background process that polling for gas prices should begin
+      gasEstimatePollToken = await getGasFeeEstimatesAndStartPolling();
+      const {
+        metamask: { gasFeeEstimates },
+      } = thunkApi.getState();
+
+      basicEstimateStatus = BASIC_ESTIMATE_STATES.READY;
```

Comment:
kind of hacky using a constnat from another slice but we need to know this for other parts of this method.